client/components/view-account.jsx

```diff
@@ -9,7 +9,8 @@ export default class ViewAccount extends React.Component {
     this.state = {
       currentRating: null,
       sidebar: false,
-      userLoggedIn: JSON.parse(window.localStorage.getItem('account'))
+      userLoggedIn: JSON.parse(window.localStorage.getItem('account')),
```

Comment:
usually, we try to do things like read from localStorage in `componentDidMount`, but it would probably be best in your app to actually pass the user info from App either as a prop to the other pages or via context